src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -246,10 +254,19 @@ const formConfig = {
           //  bit heavy-handed.
           path: 'education-history/education-information',
           uiSchema: {
```

Comment:
Potentially, but the urls are different and there are potentially some changes to date widgets that might make the pages more different than they are now. So I figured it was worth waiting, especially since most of the page is already shared.